---
layout: page
title: Tripal Citations
---

The following publications indicate use of Tripal in creation of online genome databases, development of extension modules, citation of extension modules, support in 3rd party tools, or as a recommended platform.

# 2024

- Mihai, C. A., Bădulescu, L., Asănică, A., & Iordachescu, M. (2024). Looking in the Scaffold 22 Hotspot for Differentially Regulated Genes Genomic Sequence Variation in Romanian Blueberry Cultivars. Horticulturae, 10. [http://doi.org/10.3390/horticulturae10020157](https://www.mdpi.com/2311-7524/10/2/157)

# 2023

- Volk, G., Gmitter, F., & Krueger, R. (2023). Conserving Citrus Diversity: From Vavilov’s Early Explorations to Genebanks around the World. Plants, 12. [http://doi.org/10.3390/plants12040814](https://www.mdpi.com/2223-7747/12/4/814)

- Shrestha, N., Zhang, K., Gowda, A., Abdelraheem, A., Jones, D., & Kuraparthy, V. (2023). Identification of quantitative trait loci for fiber quality, yield, and plant height traits in Upland cotton. Crop Science. [https://doi.org/10.1002/csc2.20937](https://acsess.onlinelibrary.wiley.com/doi/abs/10.1002/csc2.20937)

- Dipta, B., Sood, S., Devi, R., Bhardwaj, V., Mangal, V., Thakur, A., et al. (2023). Digitalization of potato breeding program: Improving data collection and management. Heliyon, 9, e12974. [https://doi.org/10.1016/j.heliyon.2023.e12974](https://www.sciencedirect.com/science/article/pii/S2405844023001810)

- Lipinska, A. P., Krueger-Hadfield, S. A., Godfroy, O., Dittami, S. M., Ayres-Ostrock, L., Bonthond, G., et al. (2023). "The Rhodoexplorer Platform for Red Algal Genomics and Whole-Genome Assemblies for Several Gracilaria Species". Genome Biology And Evolution, 15, evad124. [http://doi.org/10.1093/gbe/evad124](https://doi.org/10.1093/gbe/evad124) (Original work published 07AD)

- Clarke, J., Cooper, L., Poelchau, M., Berardini, T., Elser, J., Farmer, A., et al. (2023). "Data sharing and ontology use among agricultural genetics, genomics, and breeding databases and resources of the Agbiodata Consortium". Database, 2023, baad076. [http://doi.org/10.1093/database/baad076](https://doi.org/10.1093/database/baad076) (Original work published 11AD)

# 2022

- Wang, Y., Wu, J., Yan, J., Guo, M., Xu, L., Hou, L., & Zou, Q. (2022). Comparative genome analysis of plant ascomycete fungal pathogens with different lifestyles reveals distinctive virulence strategies. Bmc Genomics, 23, 1-11. ]http://doi.org/10.1186/s12864-021-08165-1](https://link.springer.com/article/10.1186/s12864-021-08165-1) (Original work published jan)

- Rolling, W. (2022). CarrotOmics: a genetics and comparative genomics database for carrot (Daucus carota). Database, 2022. [http://doi.org/10.1093/DATABASE/BAAC079](https://academic.oup.com/database/article/doi/10.1093/database/baac079/6693759) (Original work published 9AD)

# 2021

- Jung, S., Cheng, C. -H., Buble, K., Lee, T., Humann, J., Yu, J., et al. (2021). Tripal MegaSearch: a tool for interactive and customizable query and download of big data. Database, 2021. [http://doi.org/10.1093/database/baab023](https://doi.org/10.1093/database/baab023) (Original work published 04AD)

- Staton, M., Cannon, E., Sanderson, L. -A., Wegrzyn, J., Anderson, T., Buehler, S., et al. (2021). Tripal, a community update after 10 years of supporting open source, standards-based genetic, genomic and breeding databases. Briefings In Bioinformatics. [http://doi.org/10.1093/bib/bbab238](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab238/6318561) (Original work published 07AD)

- Garcia, D., Wang, Z., Guan, J., Yin, L., Geng, S., Li, A., & Mao, L. (2021). WheatGene: A genomics database for common wheat and its related species. The Crop Journal, 9, 1486-1491. [http://doi.org/10.1016/j.cj.2021.04.011](https://linkinghub.elsevier.com/retrieve/pii/S2214514121001161) (Original work published dec)

- Sanderson, L. -A., Caron, C. T., Tan, R. L., & Bett, K. E. (2021). A PostgreSQL Tripal solution for large-scale genotypic and phenotypic data . Database, 2021(baab051). [https://doi.org/10.1093/database/baab051](https://academic.oup.com/database/article/doi/10.1093/database/baab051/6352209) (Original work published 14 August 2021AD)

- Skern-Mauritzen, R., Malde, K., Eichner, C., Dondrup, M., Furmanek, T., Besnier, F., et al. (2021). The salmon louse genome: Copepod features and parasitic adaptations. Genomics, 113, 3666-3680. [http://doi.org/10.1016/j.ygeno.2021.08.002](https://linkinghub.elsevier.com/retrieve/pii/S0888754321003098) (Original work published nov)

# 2020

- Condon, B., Almsaeed, A., Buehler, S., Childers, C., Ficklin, S., Staton, M., & Poelchau, M. (2020). Tripal EUtils: a Tripal module to increase exchange and reuse of genome assembly metadata. Database, 2020. http://doi.org/10.1093/database/baz143 (Original work published 01AD)
- https://doi.org/10.1093/database/baz143

- Spoor, S., Wytko, C., Soto, B., Chen, M., Almsaeed, A., Condon, B., et al. (2020). Tripal and Galaxy: supporting reproducible scientific workflows for community biological databases. Database, 2020. http://doi.org/10.1093/database/baaa032 (Original work published 07AD)
- https://doi.org/10.1093/database/baaa032

- Gong, Y., Kang, N., Kim, Y., Wang, Z., Wei, L., Xin, Y., et al. (2020). The NanDeSyn database for Nannochloropsis systems and synthetic biology. The Plant Journal, 104, 1736-1745. http://doi.org/10.1111/TPJ.15025 (Original work published dec)
- https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.15025 https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.15025 https://onlinelibrary.wiley.com/doi/10.1111/tpj.15025

- Gui, S., Yang, L., Li, J., Luo, J., Xu, X., Yuan, J., et al. (2020). ZEAMAP, a Comprehensive Database Adapted to the Maize Multi-Omics Era. Iscience, 23, 101241. http://doi.org/10.1016/j.isci.2020.101241 (Original work published jun)
- https://linkinghub.elsevier.com/retrieve/pii/S2589004220304260

# 2019

- Peace, C. P., Bianco, L., Troggio, M., van de Weg, E., Howard, N. P., Cornille, A., et al. (2019). Apple whole genome sequences: recent advances and new prospects. Horticulture Research, 6, 59. http://doi.org/10.1038/s41438-019-0141-7 (Original work published dec)
- http://www.nature.com/articles/s41438-019-0141-7

- Jung, S., Lee, T., Cheng, C. H., Buble, K., Zheng, P., Yu, J., et al. (2019). 15 years of GDR: New data and functionality in the Genome Database for Rosaceae. Nucleic Acids Research, 47, D1137—D1145. http://doi.org/10.1093/nar/gky1000
- https://academic.oup.com/nar/article/47/D1/D1137/5144131

- Zheng, Y., Wu, S., Bai, Y., Sun, H., Jiao, C., Guo, S., et al. (2019). Cucurbit Genomics Database (CuGenDB): A central portal for comparative and functional genomics of cucurbit crops. Nucleic Acids Research. http://doi.org/10.1093/nar/gky944
- https://academic.oup.com/nar/article/47/D1/D1128/5128937

# 2018

- Falk, T., Herndon, N., Grau, E., Buehler, S., Richter, P., Zaman, S., et al. (2018). Growing and cultivating the forest genomics database, TreeGenes. Database, 2018, 1-11. http://doi.org/10.1093/database/bay084
- https://www.ncbi.nlm.nih.gov/pmc/PMC6146132/

- Condon, B., Almsaeed, A., Chen, M., West, J., & Staton, M. (2018). Tripal Developer Toolkit. Database, 2018. http://doi.org/10.1093/database/bay099
- https://dx.doi.org/10.1093/database/bay099

- Moreno, L. F., Vicente, V. A., & De Hoog, S. (2018). Black yeasts in the omics era: Achievements and challenges. Medical Mycology, 56, S32—S41. http://doi.org/10.1093/mmy/myx129 (Original work published apr)
- https://academic.oup.com/mmy/article/56/suppl\_1/S32/4925973

- Harper, L., Campbell, J., Cannon, E. K. S., Jung, S., Poelchau, M., Walls, R., et al. (2018). AgBioData consortium recommendations for sustainable genomics and genetics databases for agriculture. Database, 2018, bay088—bay088. http://doi.org/10.1093/database/bay088
- http://dx.doi.org/10.1093/database/bay088

# 2017

- Ruas, M., Guignon, V., Sempere, G., Sardos, J., Hueber, Y., Duvergey, H., et al. (2017). MGIS: managing banana (Musa spp.) genetic resources information and high-throughput genotyping data. Database, 2017. http://doi.org/10.1093/database/bax046 (Original work published jan)
- https://academic.oup.com/database/article/doi/10.1093/database/bax046/3866796

- Vining, K. J., Johnson, S. R., Ahkami, A., Lange, I., Parrish, A. N., Trapp, S. C., et al. (2017). Draft Genome Sequence of Mentha longifolia and Development of Resources for Mint Cultivar Improvement. Molecular Plant, 10, 323-339. http://doi.org/https://doi.org/10.1016/j.molp.2016.10.018
- https://www.sciencedirect.com/science/article/pii/S1674205216302659

- Watts, N. A., & Feltus, F. A. (2017). Big Data Smart Socket (BDSS): A system that abstracts data transfer habits from end users. Bioinformatics, 33, 627-628. http://doi.org/10.1093/bioinformatics/btw679
- http://dx.doi.org/10.1093/bioinformatics/btw679

- Wytko, C., Soto, B., & Ficklin, S. P. (2017). Blend4php: A PHP API for galaxy. Database, 2017, baw154—baw154. http://doi.org/10.1093/database/baw154
- http://dx.doi.org/10.1093/database/baw154

- Chen, M., Henry, N., Almsaeed, A., Zhou, X., Wegrzyn, J., Ficklin, S., & Staton, M. (2017). New extension software modules to enhance searching and display of transcriptome data in Tripal databases. Database : The Journal Of Biological Databases And Curation, 2017. http://doi.org/10.1093/database/bax052
- https://academic.oup.com/database/article/doi/10.1093/database/bax052/4049442

- Vining, K. J., Johnson, S. R., Ahkami, A., Lange, I., Parrish, A. N., Trapp, S. C., et al. (2017). Draft Genome Sequence of Mentha longifolia and Development of Resources for Mint Cultivar Improvement. Molecular Plant, 10, 323-339. http://doi.org/10.1016/j.molp.2016.10.018 (Original work published feb)
- https://www.sciencedirect.com/science/article/pii/S1674205216302659?via\%3Dihub

- Jung, S., Lee, T., Cheng, C. -H., Ficklin, S., Yu, J., Humann, J., & Main, D. (2017). Extension modules for storage, visualization and querying of genomic, genetic and breeding data in Tripal databases. Database, 2017. http://doi.org/10.1093/database/bax092 (Original work published jan)
- https://academic.oup.com/database/article/doi/10.1093/database/bax092/4718480

- Andrews, R. J., Baber, L., & Moss, W. N. (2017). RNAStructuromeDB: A genome-wide database for RNA structural inference. Scientific Reports, 7, 17269. http://doi.org/10.1038/s41598-017-17510-y (Original work published dec)
- http://www.nature.com/articles/s41598-017-17510-y

- Nazzicari, N., Caprera, A., Rossini, L., Tartarini, S., Dondini, L., Patocchi, A., et al. (2017). FruitBreedomics phenotypes and genotypes database and tools. Acta Horticulturae, 1172, 429-434. http://doi.org/10.17660/ActaHortic.2017.1172.81 (Original work published sep)
- https://www.actahort.org/books/1172/1172\_81.htm

# 2016

- Jung, S., Lee, T., Ficklin, S., Yu, J., Cheng, C. H., & Main, D. (2016). Chado use case: Storing genomic, genetic and breeding data of Rosaceae and Gossypium crops in Chado. Database, 2016. http://doi.org/10.1093/database/baw010
- https://academic.oup.com/database/article/doi/10.1093/database/baw058/2630377

- Dash, S., Campbell, J. D., Cannon, E. K. S., Cleary, A. M., Huang, W., Kalberer, S. R., et al. (2016). Legume information system (LegumeInfo.org): A key component of a set of federated data resources for the legume family. Nucleic Acids Research, 44, D1181—D1188. http://doi.org/10.1093/nar/gkv1159
- https://dx.doi.org/10.1093/nar/gkv1159

# 2015

- Krishnakumar, V., Hanlon, M. R., Contrino, S., Ferlanti, E. S., Karamycheva, S., Kim, M., et al. (2015). Araport: The Arabidopsis Information Portal. Nucleic Acids Research, 43, D1003—D1009. http://doi.org/10.1093/nar/gku1200 (Original work published jan)
- http://academic.oup.com/nar/article/43/D1/D1003/2439074/Araport-the-Arabidopsis-Information-Portal

- Dereeper, A., Bocs, S. \ ephanie, Rouard, M., Guignon, V., Ravel, S. \ ebastien, Tranchant-Dubreuil, C., et al. (2015). The coffee genome hub: A resource for coffee genomes. Nucleic Acids Research, 43, D1028—D1035. http://doi.org/10.1093/nar/gku1108 (Original work published jan)
- http://academic.oup.com/nar/article/43/D1/D1028/2437520/The-coffee-genome-hub-a-resource-for-coffee

- Krishnakumar, V., Kim, M., Rosen, B. D., Karamycheva, S., Bidwell, S. L., Tang, H., & Town, C. D. (2015). MTGD: The medicago truncatula genome database. Plant And Cell Physiology, 56, e1. http://doi.org/10.1093/pcp/pcu179 (Original work published jan)
- https://academic.oup.com/pcp/article-lookup/doi/10.1093/pcp/pcu179

- Poelchau, M., Childers, C., Moore, G., Tsavatapalli, V., Evans, J., Lee, C. Y., et al. (2015). The i5k Workspace@NAL-enabling genomic data access, visualization and curation of arthropod genomes. Nucleic Acids Research, 43, D714—D719. http://doi.org/10.1093/nar/gku983
- http://nar.oxfordjournals.org/content/43/D1/D714.abstract

# 2014

- Yu, J., Jung, S., Cheng, C. H., Ficklin, S. P., Lee, T., Zheng, P., et al. (2014). CottonGen: A genomics, genetics and breeding database for cotton research. Nucleic Acids Research, 42, 1229-36. http://doi.org/10.1093/nar/gkt1064
- http://www.ncbi.nlm.nih.gov/pubmed/24203703

- Jung, S., Ficklin, S. P., Lee, T., Cheng, C. H., Blenda, A., Zheng, P., et al. (2014). The Genome Database for Rosaceae (GDR): Year 10 update. Nucleic Acids Research, 42, 1237-44. http://doi.org/10.1093/nar/gkt1012
- http://www.ncbi.nlm.nih.gov/pubmed/24225320

# 2013

- Ficklin, S. P., & Feltus, F. A. (2013). A Systems-Genetics Approach and Data Mining Tool to Assist in the Discovery of Genes Underlying Complex Traits in Oryza sativa. Plos One, 8, e68551. http://doi.org/10.1371/journal.pone.0068551
- http://www.ncbi.nlm.nih.gov/pubmed/23874666

- Droc, G. etan, ere, D. L. \, Guignon, V., Yahiaoui, N., This, D., Garsmeur, O., et al. (2013). The banana genome hub. Database, 2013. http://doi.org/10.1093/database/bat035
- http://database.oxfordjournals.org/content/2013/bat035.abstract

# 2012

- Wegrzyn, J. L., Main, D., Figueroa, B., Choi, M., Yu, J., Neale, D. B., et al. (2012). Uniform standards for genome databases in forest and fruit trees. Tree Genetics And Genomes, 8, 549-557. http://doi.org/10.1007/s11295-012-0494-7
- https://link.springer.com/article/10.1007/s11295-012-0494-7

# 2011

- Jung, S., Menda, N., Redmond, S., Buels, R. M., Friesen, M., Bendana, Y., et al. (2011). The Chado Natural Diversity module: A new generic database schema for large-scale phenotyping and genotyping data. Database, 2011, bar051—bar051. http://doi.org/10.1093/database/bar051
- https://www.ncbi.nlm.nih.gov/pmc/PMC3225077/
