---
title: Tripal
subtitle: An open-source toolkit for construction of genomic, genetic, breeding and biological databases.
layout: frontpage
---

Genomics, genetics, breeding and other biological data are increasingly complicated and time consuming to publish online for other researchers to search, browse and make discoveries. Tripal provides a framework to reduce the complexity of creating such a site, and provides access to a community of similar groups that share community-standards, and interact to address questions and learn best practices for sharing, storing and visualizing complex biological data.

# Objectives

- Provide a framework for those with genomic, genetic and breeding data for creation of an online site for display, search and visualization.
- To use community-derived standards and ontologies to facility continuity between sites which in turn fosters collaboration and sharing
- Provide an out-of-the-box setup for a genomics site for those who simply want to put new genome assemblies and annotations online.
- Provide Application Programming Interfaces (APIs) for complete customization such that more advanced displays, look-and-feel, and new functionality can be supported.

# Major Features

- Incorporates the [GMOD Chado database schema](https://gmod.org/wiki/Chado_-_Getting_Started) which serves as the data warehouse.
- Integrates with Drupal for user management, and non-biological content.
- Data loaders for commonly used file formats (FASTA, GFF3, GAF, OBO, NCBI Taxonomy, PubMed, newick, tab-delimited, etc.).
- Sites can be customized as desired.
- Site developers can create new extension modules to share with other Tripal sites.

# To Cite Tripal

To Cite Tripal please use one of the following publications:

 - Spoor S, Cheng CH, Sanderson LA, Condon B, Almsaeed A, Chen M, Bretaudeau A, Rasche H, Jung S, Main D, Bett K, Staton M, Wegrzyn JL, Feltus FA, Ficklin SP. [Tripal v3: an ontology-based toolkit for construction of FAIR biological community databases](https://academic.oup.com/database/article-lookup/doi/10.1093/database/baz077). Database. July 2019, 2019: baz077.
 - Lacey-Anne Sanderson, Stephen P. Ficklin, Chun-Huai Cheng, Sook Jung, Frank A. Feltus, Kirstin E. Bett, and Dorrie Main. [Tripal v1.1: a standards-based toolkit for construction of online genetic and genomic databases](https://database.oxfordjournals.org/content/2013/bat075.short). Database, Sept 2013: bat075.
 - Stephen P. Ficklin, Lacey-Anne Sanderson, Chun-Huai Cheng, Margaret Staton, Taein Lee, Il-Hyung Cho, Sook Jung, Kirstin E Bett, Dorrie Main. [Tripal: a construction Toolkit for Online Genome Databases](https://database.oxfordjournals.org/content/2011/bar044.long). Database, Sept 2011: bar044.

# Brief History

Tripal was first created in 2008 at the Clemson University Genomics Institute (CUGI) by Stephen Ficklin and Meg Staton to create a searchable annotated transcriptomic websites for the marine genomics and Fagaceae (hardwood) communities. It was created to help lower the total cost of development for these two communities, but with the goal to create a tool that could be re-used, was expandable, modular, provided support for social needs of a research community, did not force any particular look-and-feel and allowed for customizations as needed by the community. The first official release of Tripal occurred in 2009 and Tripal was soon added as a member of the [Generic Model Organism Database (GMOD)](https://www.gmod.org/) organization suite of genome tools. Later, Tripal was adopted by Lacey-Anne Sanderson in Dr. Kirstin Bettâ€™s lab at the University of Saskatchewan for the [KnowPulse database for pulse crop breeding and genetics](https://knowpulse.usask.ca) and Dorrie Main at Washington State University for the [Genome Database for Rosaceae (GDR)](https://www.rosaceae.org/). Lacey Sanderson of Dr. Bett's group provided major design and developmental support for Tripal. These two groups continued collaborative development of Tripal adding support for whole genome data, genetic data, publications, data for large-scale genotyping and phenotyping data, and data loaders for common formats. Currently, both groups continue active development, provide support to the community and are actively creating new resources.
